test/controllers/clinics_controller_test.rb

```diff
@@ -21,20 +22,22 @@ class ClinicsControllerTest < ActionDispatch::IntegrationTest
   describe 'create' do
     before { @new_clinic = attributes_for :clinic }
 
-    # it 'should raise if user is not admin' do # TODO
-    #   @user.role = :cm
-    #   @user.save!
-    #   assert_raise RuntimeError, 'Permission Denied' do
-    #     post :create
-    #   end
-    # end
+    it 'should raise if user is not admin' do
```

Comment:
I don't think this is raising anything. Wrong name?

Reply:
yep, good catch! retitling this test to reflect the redirection.